documentation/MeetingMinutes/TSC/TSC-2023-10-19-Meeting-Minutes.md

```diff
@@ -64,14 +64,15 @@ Following points were discussed:
 **Discussion on API versioning** – DT proposal to use alpha<n> in API path until and API (major) version is stable
   * Distinction between repostiory release (semantic version as before) and the version shown within API base path
     * A sub project repository can contain one or multiple (closely related) APIs
-    * The version number of releases on sub project level will be used within all APIs within the repository (= the API release belongs to vX.Y.Z of the sub project)
+    * API has not to have the same version that the family release but release will match the most advanced API in sub-project. E.g.: Release 3.0 with API-X in version 3.0, API-Y in version v2alpha1 plus in version v1.0.2, etc)

```

Comment:
According to my comment above I propose the following changed sentence (but also propose to have further discussion on a detailed concept):

* APIs within a sub project will all belong to the sub project release, but can have have different major versions within the base path. The sub project release will match the major version of the most advanced stable API in the sub-project. E.g.: Release version will be 3.x.y if there is API-X in version v3 together with API-Y in v1 and v2alpha2.

Reply:
Fine for me